src/sentry/static/sentry/app/utils/eventWaiter.tsx

```diff
@@ -9,7 +9,7 @@ import withApi from 'app/utils/withApi';
 const DEFAULT_POLL_INTERVAL = 5000;
 
 const recordAnalyticsFirstEvent = ({organization, project}) =>
-  analytics('onboarding_v2.first_event_recieved', {
+  analytics('onboarding_v2.first_event_received', {
```

Comment:
We should double check this matches reload or not

Reply:
Unfortunately it does.

https://github.com/getsentry/reload/blob/7e63a57a8e411753bfbfd66de6ffc52fb5bf0fbb/reload_app/events.py#L630

This will have to be left misspelled